_maps/RandomZLevels/beach.dmm

```diff
@@ -253,7 +253,7 @@
 /obj/machinery/gateway{
 	dir = 9
 	},
-/obj/effect/turf_decal/sand/warning{
+/obj/effect/turf_decal/sand{
 	dir = 9
```

Comment:
However you kept the dir's on all of these.

sand/warning did have different patterns too compared to the base sandyfloor & sandyplating sprites, so I'm not crazy about them being removed.